crypto_sign/dilithium2/clean/symmetric.h

```diff
@@ -0,0 +1,41 @@
+#ifndef SYMMETRIC_H
+#define SYMMETRIC_H
+
+#include "fips202.h"
+#include "params.h"
+
+typedef struct {
+    uint64_t s[25];
+} keccak_state;
+
+void PQCLEAN_DILITHIUM2_CLEAN_shake256_absorb(keccak_state *state, const uint8_t *input, size_t inlen);
+void PQCLEAN_DILITHIUM2_CLEAN_shake256_squeezeblocks(uint8_t *output, size_t nblocks, keccak_state *state);
+void PQCLEAN_DILITHIUM2_CLEAN_shake128_absorb(keccak_state *state, const uint8_t *input, size_t inlen);
+void PQCLEAN_DILITHIUM2_CLEAN_shake128_squeezeblocks(uint8_t *output, size_t nblocks, keccak_state *state);
+#define PQCLEAN_DILITHIUM2_CLEAN_shake256_absorb(STATE, IN, INBYTES) (shake256_absorb((uint64_t *) (STATE), IN, INBYTES))
+#define PQCLEAN_DILITHIUM2_CLEAN_shake256_squeezeblocks(OUTPUT, OUTBLOCKS, STATE) (shake256_squeezeblocks(OUTPUT, OUTBLOCKS, (uint64_t *) (STATE)))
+#define PQCLEAN_DILITHIUM2_CLEAN_shake128_absorb(STATE, IN, INBYTES) (shake128_absorb((uint64_t *) (STATE), IN, INBYTES))
+#define PQCLEAN_DILITHIUM2_CLEAN_shake128_squeezeblocks(OUTPUT, OUTBLOCKS, STATE) (shake128_squeezeblocks(OUTPUT, OUTBLOCKS, (uint64_t *) (STATE)))
```

Comment:
do you still need these to be defined if you reimplement `shake256`  in terms of #175 ?

Reply:
nope - I removed those.